nibabel/volumeutils.py

```diff
@@ -1345,7 +1346,8 @@ def _ftype4scaled_finite(tst_arr, slope, inter, direction='read',
 
 
 def finite_range(arr, check_nan=False):
-    ''' Return range (min, max) or range and flag (min, max, has_nan) from `arr`
+    ''' Return range (min, max) or range and flag (min, max, has_nan) from
```

Comment:
Should be on single line for numpy docstring standard.  Maybe ``Get range (min, max) or range and flag (min, max, has_nan) from `arr```